src/main/java/uk/gov/justice/digital/delius/service/TierService.java

```diff
@@ -33,7 +33,6 @@ public class TierService {
     private final ContactService contactService;
     private final StaffRepository staffRepository;
     private final TeamRepository teamRepository;
-    private final SpgNotificationService spgNotificationService;
```

Comment:
Why has this service and the call to it been removed?

Reply:
https://github.com/ministryofjustice/community-api/pull/706#discussion_r948167991

Reply:
😄